index.html

```diff
@@ -1,140 +1,170 @@
 				</div>
-				<div class="stud"></div>
-				<h2 id="contribute">Contribute</h2>
-				<div class="content contributecontent">
-					<div class="vcentercontainer detailsimg">
-						<div>
-							<img src="img/details_features.png" alt="BASH Drawing">
-							<img src="img/details_contribute.png" alt="Git for Windows Logo">
-						</div>
+			</div>
+		</div>
+	</section>
+	<section>
+		<div class="container">
+			<h2>Tools &amp; Features</h2>
+			<p class="text-muted mb-5">Git for Windows focuses on offering a lightweight, native set of tools that bring
+				the full feature set of the <a href="https://git-scm.com" target="_blank">Git SCM</a> to Windows while
+				providing appropriate user interfaces for experienced Git users and novices alike.</p>
+			<div class="row">
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw1web_thumb.png" alt="Git BASH" class="img-other-logos">
+					<h3>Git BASH</h3>
+					<p>Git for Windows provides a BASH emulation used to run Git from the command line. *NIX users
+						should feel right at home, as the BASH emulation behaves just like the "git" command in LINUX
+						and UNIX environments.</p>
+				</div>
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw2web_thumb.png" alt="Git GUI" class="img-other-logos">
+					<h3>Git GUI</h3>
+					<p>As Windows users commonly expect graphical user interfaces, Git for Windows also provides the Git
+						GUI, a powerful alternative to Git BASH, offering a graphical version of just about every Git
+						command line function, as well as comprehensive visual diff tools.</p>
+				</div>
+				<div class="col-sm-6 col-lg-4 mb-3">
+					<img src="img/gw3web_thumb.png" alt="Gitk: Exploring Commits" class="img-other-logos">
+					<h3>Shell Integration</h3>
+					<p>Simply right-click on a folder in Windows Explorer to access the BASH or GUI.</p>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="bg-light" id="contribute">
+		<div class="container">
+			<h2>Contribute</h2>
+			<p>As an open source project, Git for Windows benefits greatly from both the volunteer work of helpful
+				developers and <a href="https://github.com/git-for-windows/git/wiki/Issue-reporting-guidelines">good bug
+					reports</a> made by Git for Windows' users.</p>
+			<h3>Governance Model</h3>
+			<p>If you are interested in contributing to the <i>Git for Windows</i> project, have a look at the current
+				<a href="governance-model.html">governance model</a>. There the roles of those involved in driving this
+				project further down the road are described in detail.</p>
+			<h3><a name="issues"></a>Bug Reports &amp; Feature Requests</h3>
+			<p>If you've noticed a bug or simply have an idea that you'd like to see become real, why not work on it?
+				Bug reports and feature requests are typically submitted to the issue tracker on our GitHub repository.
+			</p>
+			<a class="btn btn-primary shadow mr-2" href="https://github.com/git-for-windows/git/issues"
```

Comment:
I don't understand why this is a button orphaned below a paragraph.

<img width="217" alt="orphan" src="https://user-images.githubusercontent.com/714283/142431579-2f4fa3e0-0fe5-4b46-9074-c2998c37e486.png">

It would look nicer to fold it into the paragraph and de-button-ify it. (That said, I see merit in having a button link to the issue tracker somewhere on the page - perhaps with other buttons so it's not out of place?)

Reply:
I also don't understand why it's a `btn-primary` when there's already a `btn-primary` on the page. If a button remains, it should be `btn-outline-primary` to reduce its emphasis (it's not the core CTA on the page).

Reply:
@vtbassmatt I agree but the question here. is if we are already providing an issue tracker like on a navbar like the FAQ and others, why keep a Bug Reports section on the website at all?

Reply:
@prertik that's a fine question! Looking at another friendly project, [Django](https://djangoproject.com), I see that they have a header with "Community", "Code", and "Issues" explicitly separated. (This makes sense since they use a mailing list/IRC, GitHub, and some legacy bug tracker respectively.)

<img width="748" alt="Django" src="https://user-images.githubusercontent.com/714283/142444422-baf95a6f-f719-4687-a9b1-9c594383de31.png">

Then way down the page, in the sidebar, they use a very short section of the column to repeat their connections to community-oriented stuff.

<img width="409" alt="image" src="https://user-images.githubusercontent.com/714283/142444581-6584b98b-c18b-4213-ad38-e14ce52b8c7b.png">

I think we could probably live without the large paragraph here, if you're keen to remove it.

Reply:
Thanks for explaining! Okay, I'll try this and see how this looks.